erpnext/stock/doctype/item/item.js

```diff
@@ -26,19 +26,19 @@ frappe.ui.form.on("Item", {
 
 	refresh: function(frm) {
 		if (frm.doc.is_stock_item) {
-			frm.add_custom_button(__("Balance"), function() {
+			frm.add_custom_button(__("Stock Balance"), function() {
 				frappe.route_options = {
 					"item_code": frm.doc.name
 				}
 				frappe.set_route("query-report", "Stock Balance");
 			}, __("View"));
-			frm.add_custom_button(__("Ledger"), function() {
+			frm.add_custom_button(__("Stock Ledger"), function() {
 				frappe.route_options = {
 					"item_code": frm.doc.name
 				}
 				frappe.set_route("query-report", "Stock Ledger");
 			}, __("View"));
-			frm.add_custom_button(__("Projected"), function() {
+			frm.add_custom_button(__("Stock Projected Qty"), function() {
```

Comment:
Change the "Stock Projected Qty" to "Stock Projected Quantity", because everywhere else we don't use short forms.

Reply:
The reason I named it Stock Projected Qty is because the report itself is called "Stock Projected Qty". Should we change the report name to "Stock Projected Quantity"?

Reply:
There are too many references to the report.
Let's keep it as is.